components/title/basic/src/index.js

```diff
@@ -0,0 +1,51 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+/**
+ * Builds the modifier className to apply to the component following the SUIT naming convention (sui-TitleBasic--modifierInCamelCase).
+ * @param alignProp {string} property value for Title alignment.
+ * @returns {string} SUIT formatted classname.
+ */
+const _buildAlignClassName = (alignProp) => {
+  return alignProp ? 'sui-TitleBasic--align' + alignProp.charAt(0).toUpperCase() + alignProp.slice(1) : ''
+}
+
+export default function TitleBasic ({align, className, subtitle, title}) {
+  const titleBasicClassName = cx('sui-TitleBasic', className, `${_buildAlignClassName(align)}`)
+
+  return (
+    <div className={titleBasicClassName}>
+      {title && <h1 className={`sui-TitleBasic-title`}>{title}</h1>}
+      {subtitle && <h2 className={`sui-TitleBasic-subtitle`}>{subtitle}</h2>}
+    </div>
+  )
+}
+
+TitleBasic.displayName = 'TitleBasic'
+
+TitleBasic.propTypes = {
+  /**
+   * Defines the text alignment of title and subtitle.
+   */
+  align: PropTypes.oneOf([
+    'left',
+    'center',
+    'right'
+  ]),
+  /**
+   * CSS classNames to apply to Title Basic container.
+   */
+  className: PropTypes.string,
+  /**
+   * Title to display.
+   */
+  title: PropTypes.string.isRequired,
```

Comment:
I agree with this, it is a bit redundant, so I remove the validation in render function.